estate/models/estate_property.py

```diff
@@ -0,0 +1,47 @@
+"""docstring estate property"""
```

Comment:
This docstring does not carry much value, if it is your linter's idea you should try to disable this specific rule

Reply:
It is my linter's idea. There's a few of these rules I should disable, tbh